priv/www/nitrogen.js

```diff
@@ -1221,16 +1221,25 @@ NitrogenClass.prototype.$sortblock = function(el, sortOptions, sortPostbackInfo)
 }
 
 /*** TEXTAREA TAB-TRAPPING ***/
-/*** With a little help from http://stackoverflow.com/questions/6140632/how-to-handle-tab-in-textarea ***/
-NitrogenClass.prototype.$trap_tabs = function(el) {
-    $(el).keydown(function(e) {
+/**
+ * Intercept tab keypresses and insert a tab character.
+ * 
+ * Don't do this, as it will interferes with accessibility APIs.
+ * 
+ * See http://stackoverflow.com/questions/6140632/how-to-handle-tab-in-textarea
+ * 
+ * @param {string} Unique class name anchoring textarea.
+ */
+NitrogenClass.prototype.$trap_tabs = function(textarea_class_name) {
+    var textarea = document.querySelector(el);
```

Comment:
Only issue here is just a typo.  The passed arg is `textarea_class_name`, while the querySelector is using `el`. As you suspected, this function is only really being called with a selector string, so we can rely on that.

I've fixed this in my branch, and I'm working on the merge hopefully for today.

Reply:
Thanks. I must have been distracted when I pushed it out. Also I found a typo in the docblock comments, whereas the argument name is missing from the `@param` line